plugins/Submit/templates/displayForm;submit;default

```diff
@@ -70,6 +70,9 @@ __template__
 			<div class="article">
 				[% stripped_title = form.subj | strip_title %]
 				[% PROCESS titlebar width => "100%", title => stripped_title %]
+			<div class="details">
+				[% IF constants.use_dept && constants.submit_dept && form.dept %]<strong>from the <b>[% form.dept %]</b> dept.</strong>[% END %]
```

Comment:
We should have some kind of a strip_ before we display form.dept. I'd need to refresh my memory on them as to which though.